flow/templates/expanse.sh

```diff
@@ -49,7 +48,7 @@
     {% endif %}
 {% endblock tasks %}
 {% block header %}
-    {{ super() -}}
+    {{- super() -}}
```

Comment:
For future reference, this is the important fix. The dash here will remove leading whitespace from the output of the `super()` call. This matches other templates.